VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.ts

```diff
@@ -99,6 +102,17 @@ export class ParticipantWaitingRoomComponent extends WaitingRoomBaseComponent im
             this.checkIfHearingIsClosed();
             this.checkIfHearingIsStarting();
         });
+        this.closedSubscription$ = this.clockService.getClock().subscribe(time => {
```

Comment:
remove this & just use the clock subscription above - you're creating two subscriptions to the same obseravble by doing this

Reply:
Done